src/main/java/com/google/cloud/teleport/v2/bigquery-gcs/src/main/java/com/google/cloud/teleport/v2/bigquerygcs/BigQueryToParquet.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.bigquerygcs;
```

Comment:
Change this package name to:
com.google.cloud.teleport.v2.templates;

Reply:
@sabhyankar okay will do. So what I think your looking for is the following structure:
```
v2/
     pom.xml <-- this is the parent pom
     checkstyle/
     bigquery-to-parquet/  <----- this is a module.
                        pom.xml <---- this is the child pom.xml for this module
                        src/main/java/com/google/cloud/teleport/v2/templates/BigQueryToParquet.java
                        src/main/resources/<command_spec>
                        src/test/java/com/google/cloud/teleport/v2/templates/BigQueryToParquet.java
```

Reply:
Each template will be a module.